spec/form_models/publishers/job_listing/job_details_form_spec.rb

```diff
@@ -7,8 +7,6 @@
   it { is_expected.to allow_value("Job &amp; another job").for(:job_title) }
   it { is_expected.not_to allow_value("Title with <p>tags</p>").for(:job_title).with_message(I18n.t("job_details_errors.job_title.invalid_characters")) }
 
-  it { is_expected.to validate_presence_of(:working_patterns) }
-
   # it { is_expected.to validate_inclusion_of(:contract_type).in_array(Vacancy.contract_types.keys) }
```

Comment:
I wonder why this test is commented out. When I run it, it passes. Could you uncomment it please? (I know it wasn't you.)

Reply:
@josephhull676 I feel like you asked that just the other week and I didn't have an answer for you 🤣 

Agree we should just uncomment it.

Reply:
Sweet! 

And yes you did @csutter 😂